tests/cloudlaunchprocess/test.csv

```diff
@@ -0,0 +1,27 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+comment,0,,"Test 1 - Launch and then kill a process with all possible cmd line args"
+before,0,0,q,discport:1+"I"$getenv[`KDBBASEPORT] ,,"Find the port of the discovery process"
+run,0,0,q,launch input1,,"Start a new process with extra command line args"
+run,0,0,q,system "sleep 3",,"Wait for process to start"
+run,0,0,q,h:hopen `::7124:admin:admin ,,"Attempt to connect to the new process"
+true,0,0,q,`default=h".wdb.writedownmode",,"Check that connection has been made"
+true,0,0,q,110011011b~(2 _ (2 cut h".z.X")[;1]) in value input1,,"Check that the cmd line args are as expected "
```

Comment:
I agree that it is a bit of an ugly solution but the problem is that some values don't agree exactly between input1 and the cmd line parameters of the process. 

For example variables like ${KDBAPPCONFIG}, which has its variable name in input1, but the actual value of the variable in the cmd line parameters. Also some values in input1 end up being enlisted in the cmd line parameters and so the test isn't recognising that "1" and ,"1" are equal. 

That's why I ended up resorting to that binary string.

I will definitely use .Q.opt h".z.X" instead of my other way though.